webapp/src/ts/services/contact-view-model-generator.service.ts

```diff
@@ -164,7 +164,8 @@ export class ContactViewModelGeneratorService {
       .then(doc => {
         children.push({
           doc: doc,
-          isPrimaryContact: true
+          isPrimaryContact: true,
+          id: doc._id,
```

Comment:
```suggestion
          id: doc?._id,
```
not sure if doc can be undefined here

Reply:
Related, line 154 null check?  `const primaryContact = children.find(child => child.doc?._id === contactId);`
but if all docs have _id 100% then can be ignored :)

Reply:
If the doc is undefined, we have biiig problems :D we shouldn't just go on with the execution. 
So this throwing an error should be fine.